src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java

```diff
@@ -127,7 +127,7 @@ public static String encodePEM(@Nonnull KeyPair keys) throws IOException {
         String stripedPEM = StringUtils.stripEnd(StringUtils.strip(pem, header), header);
         // sanity cleanup
         stripedPEM = stripedPEM.replaceAll("(\r|\n|\t| )", "");
-        return DatatypeConverter.parseBase64Binary(stripedPEM);
+        return Base64.decodeBase64(stripedPEM);
```

Comment:
Would it be better to use `java.util.Base64.getMimeDecoder().decode()` to avoid needing the commons-codec dependency?